libkbfs/repeated_wait_group.go

```diff
@@ -20,6 +20,8 @@ type RepeatedWaitGroup struct {
 	lock     sync.Mutex
 	num      int
 	isIdleCh chan struct{} // leave as nil when initializing
+	paused   bool
```

Comment:
There's currently no initializer for `RepeatedWaitGroup` and I don't feel like adding one in this PR -- that makes its usage too different from `sync.WaitGroup` in my opinion.  I'll add a TODO though.